companies-manifest.js

```diff
@@ -81,6 +82,11 @@ const companies = [
     similarWebGlobalRank: 114,
     logo: BBCLogo,
   },
+  {
+    key: 'https://www.blablacar.de',
```

Comment:
Since bla bla car has a bunch of local websites, let's use their most widely used one, which appears to be .fr! https://www.similarweb.com/fr/website/blablacar.fr

```suggestion
    key: 'https://www.blablacar.fr',
```